src/promptflow/promptflow/_sdk/operations/_run_operations.py

```diff
@@ -233,6 +233,8 @@ def delete(
         name: str,
     ) -> None:
         """Delete run permanently.
+        Caution: This operation will delete the run permanently from your local disk.
```

Comment:
seems irelevant to this PR?

Reply:
Yes, this is a modification that was missed in the previous PR, I will change it in this PR.